app/views/job_invocations/_tab_hosts.html.erb

```diff
@@ -12,7 +12,7 @@
     <tbody>
       <% job_invocation.targeting.hosts.each do |host| %>
         <tr>
-          <% task = job_invocation.sub_tasks.select { |task| task.locks.where(:resource_type => 'Host::Managed', :resource_id => host.id).any? }.first %>
+          <% task = job_invocation.sub_tasks.joins(:locks).where("#{ForemanTasks::Lock.table_name}.resource_type" => 'Host::Managed', "#{ForemanTasks::Lock.table_name}.resource_id" => host.id).first %>
```

Comment:
not sure how would I then detect right lock to get the task object for a particular host, even if I do `.include(:locks)` the condition changes the query so it would add another locks query for each host

I know I could add custom select to keep the host id in task object but that's an ugly hack... cleaner would be to define the association on host through template invocation tasks and include it directly but I think we'll replace this by task groups soon so I don't want to spend to much time on this... the bigger issue in terms of performance I think is that we miss pagination in this list